lib/rubocop/cop/style/empty_else.rb

```diff
@@ -151,9 +151,12 @@ def else_line_range(loc)
           loc.else.first_line..loc.end.first_line
         end
 
-        def base_if_node(node)
-          return node unless node.case_type? || node.elsif?
-          node.each_ancestor(:if).find { |parent| parent.loc.end } || node
+        def base_node(node)
+          return node if node.case_type?
+          return node unless node.elsif?
+          node.each_ancestor(:if, :case, :when).find do |parent|
```

Comment:
`find` implements the ability to specify a default value if there are no matches.

```ruby
node.each_ancestor(:if, :case, :when).find(->(node)) do |parent|
  parent.loc.end
end
```

https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find

Reply:
Done.